refarch-integrations/README.md

```diff
@@ -10,6 +10,7 @@ Collection of different integration which can be used as is in RefArch projects.
   s3-integration for file handling.
 - [dms-integration](./refarch-dms-integration/README.md): For CRUD operations on a dms system in specific fabasoft. Uses
   s3-integration for file handling.
+- [address-integration](./refarch-email-integration/README.md): For reading, searching and validating addresses.
```

Comment:
```suggestion
- [address-integration](./refarch-address-integration/README.md): For reading, searching and validating addresses using the LHM address service.
```

Gibt es vielleicht noch eine Referenz auf den LHM-Address-Service die Public liegt?

Reply:
Typo done.
As far as i know there isn't any other public information but not sure.